scrapers/bugzilla.py

```diff
@@ -0,0 +1,108 @@
+# SPDX-License-Identifier: GPL-3.0+
+from __future__ import unicode_literals
+
+from builtins import bytes
+
+from scrapers.base import BaseScraper
+from purview.models.bugzilla import BugzillaBug
+from purview.models.user import User
+import purview.utils.general as utils
```

Comment:
This import will be confusing after #20 is merged because I'm adding a `scraper.utils` file.